audio_service/lib/audio_service.dart

```diff
@@ -170,7 +170,7 @@ class PlaybackState {
   final int? queueIndex;
 
   /// Creates a [PlaybackState] with given field values, and with [updateTime]
-  /// defaulting to [DateTime.now()].
+  /// defaulting to [DateTime.now].
```

Comment:
If we're being pedantic, maybe this should be:

```dart
  /// Creates a [PlaybackState] with given field values, and with [updateTime]
  /// defaulting to `DateTime.now()`.
```

since we mean the expression value not the function...

Reply:
i see as equivalents:
* `DateTime.now()`
* [DateTime.now]
* [new DateTime.now]

i just preferred [DateTime.now] for having a reference to the class